requirements.yaml

```diff
@@ -0,0 +1,7 @@
+dependencies:
+  - name: mysql
+    repository: https://kubernetes-charts.storage.googleapis.com
+    version: 1.2.0
+    import-values:
+      - child: default.data
```

Comment:
I don't see child chart or values.

Reply:
I would remove this line, it is of no use, for now, would add it later if needed.

Reply:
Sure, also the parent is also not required for now I think.

Reply:
I think you removed that in the last commit. So, fine for now.